src/Tribe/Aggregator/Cron.php

```diff
@@ -440,7 +510,7 @@ public function verify_fetching_from_service() {
 		foreach ( $query->posts as $post ) {
 			$record = $records->get_by_post_id( $post );
 
-			if ( tribe_is_error( $record ) ) {
+			if ( $record === null || tribe_is_error( $record ) ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 65b1f8f775f619108f5de370ab204516</sub>